### ISSUE TITLE
jinjalint does not detect trailing coma in for loop

### ISSUE THREAD

**Issue by allan-simon** [motet-a/jinjalint#23]
jinjalint reports no syntax error with this code

```jinja2
{%
  for
    a,
    b,
  in [(1, 1), (2, 2)]
%}

{% endfor %}

```

while actual jinja reports
```
Syntax error in jinja2 template: expected token 'in', got '['
```
